### DISCUSSION TITLE
Graiax: 使用经验集锦

### DISCUSSION THREAD

**Discussion by EricTianC** [GraiaProject/Application#95] [作品展示]
应GreyElaina~~要求~~建议设立

## 这是什么
一个专门（~~其实是因为文档垃圾被迫~~）设立的Discussions，用于帮助小白构建自己的Bot。  
如果你对Graia框架的使用有一定经验，你也可以在本讨论下发表，~~帮助Graia社区越来越好~~。  

我们假设您对Python的基本用法有一定的了解，~~并会熟练的复制粘贴~~

**Comment by GreyElaina** [comment]
> 要求

这是建议啊我寻思...

**Comment by RainyNian** [comment]
# 提个醒  
# 如果你连Python都不知道  
# [点击这里](https://docs.python.org/zh-cn/3/)

**Comment by pylover7** [comment]
所以各位大佬，这个经验锦集啥时候更新呢

**Comment by hair2020** [comment]
快让我看看大佬的经验
